src/Simulation/Simulators.Tests/TestProjects/TargetedExe/TargetedExe.csproj

```diff
@@ -1,17 +1,20 @@
-<Project Sdk="Microsoft.Quantum.Sdk">
+﻿<Project Sdk="Microsoft.Quantum.Sdk">
```

Comment:
Do you know why many of the project files changed in this PR show this first line as changed? There is no obvious change to the first line of these project files that I can see.

Reply:
I believe the file encoding has changed (the first couple of non-printable characters are changed). 
`git diff main hardware_targeting src/Simulation/Simulators.Tests/TestProjects/TargetedExe/TargetedExe.csproj`